src/Common/Extensions/StringExtensions.cs

```diff
@@ -24,16 +24,16 @@ public static bool EqualsIgnoreCase(this string a, string b)
             return string.Equals(a, b, StringComparison.OrdinalIgnoreCase);
         }
 
-        public static string JoinKeyValuePairs(IDictionary<string, string> pairs, char pairSep = ' ', char kvSep = '=')
-        {
-            return string.Join(pairSep, pairs.Select(pair => pair.Key + kvSep + pair.Value));
+        public static string JoinKeyValuePairs(IDictionary<string, string> pairs, char pairSep = ' ', char kvSep = '=')	
+        {	
```

Comment:
nit: remove whitespace? `Ctrl + K + D` short cut to remove all whitespaces from a file